hooks/useDarkMode.js

```diff
@@ -13,16 +10,16 @@ export default function useDarkMode() {
   const [isDarkMode, setIsDarkMode] = useState();
 
   useEffect(() => {
-    setIsDarkMode(ThemeService.getIsDarkMode());
+    setIsDarkMode(getIsDarkMode());
   }, []);
 
   useEffect(() => {
-    ThemeService.setIsDarkMode(isDarkMode);
+    persistIsDarkMode(isDarkMode);
   }, [isDarkMode]);
 
   const toggleThemeChange = useCallback(() => {
     setIsDarkMode((prev) => !prev);
-  });
+  }, []);
```

Comment:
Do we even need `useCallback` here?

Reply:
Didn't touch that part, only fixed ESLint issue of missing hook deps.